vizro-core/src/vizro/models/_components/aggrid.py

```diff
@@ -0,0 +1,121 @@
+import logging
+from typing import Dict, List, Literal
+
+import pandas as pd
+from dash import State, dash_table, dcc, html
+
+try:
+    from pydantic.v1 import Field, PrivateAttr, validator
+except ImportError:  # pragma: no cov
+    from pydantic import Field, PrivateAttr, validator
+
+import vizro.tables as vt
+from vizro.actions._actions_utils import CallbackTriggerDict, _get_component_actions, _get_parent_vizro_model
+from vizro.managers import data_manager
+from vizro.models import Action, VizroBaseModel
+from vizro.models._action._actions_chain import _action_validator_factory
+from vizro.models._components._components_utils import _process_callable_data_frame
+from vizro.models._models_utils import _log_call
+from vizro.models.types import CapturedCallable
+
+logger = logging.getLogger(__name__)
+
+
+class AGGrid(VizroBaseModel):
```

Comment:
I think we should probably go for `AgGrid` here to match Dash.

Reply:
I find it doesn't look so nice, and the original seems more AG Grid, but let's do it to align with dash